README.md

```diff
@@ -1,73 +1,137 @@
-[![Gitter][gitter-badge]][gitter-url]
 [![Google Groups][groups-badge]][groups-url]
+[![Stackoverflow][stackoverflow-badge]][stackoverflow-url]
 
-https://tarantool.io/en/
+**A feature-rich in-memory database.** Our features set:
 
-Patch submissions and discussion of particular patches https://lists.tarantool.org/mailman/listinfo/tarantool-patches/
+* ACID
+* primary and secondary indexes
+* in-memory and disk engine
+* stored procedures
+* SQL
+* replication
+* sharding (as an external module)
+* and a wealth of modules and connectors from popular languages
 
-General development discussions https://lists.tarantool.org/mailman/listinfo/tarantool-discussions/
+Another Tarantool feature is the ability to write code in Lua, a general purpose
+programming language, right inside the database. You can write custom expiration
+strategies and any business logic, and synchronize data with other data sources.
 
-Tarantool is an in-memory database and application server.
+You can use Tarantool to replace Redis when
+you need secondary indexes, a different structure of storage data, SQL, etc.
 
-Key features of the application server:
- * 100% compatible drop-in replacement for Lua 5.1,
-   based on LuaJIT 2.1.
-   Simply use `#!/usr/bin/tarantool` instead of
-   `#!/usr/bin/lua` in your script.
- * full support for Lua modules and a rich set of
-   own modules, including cooperative multitasking,
-   non-blocking I/O, access to external databases, etc
+For more details, please see the
+[Tarantool documentation](https://www.tarantool.io/en/doc/2.2/).
 
-Key features of the database:
- * ANSI SQL, including views, joins, referential
-   and check constraints
- * MsgPack data format and MsgPack based
-   client-server protocol
- * two data engines: 100% in-memory with
-   optional persistence and an own implementation of LSM-tree, 
-   to use with large data sets
- * multiple index types: HASH, TREE, RTREE, BITSET
- * asynchronous master-master replication
- * authentication and access control
- * the database is just a C extension to the
-   application server and can be turned off
+## Getting started
 
-Supported platforms are Linux/x86 and FreeBSD/x86, Mac OS X.
+### Get hold of Tarantool
 
-Tarantool is ideal for data-enriched components of
-scalable Web architecture: queue servers, caches,
-stateful Web applications.
+* Try online:
 
-To download and install Tarantool as a binary package for your OS, please visit
-https://tarantool.io/en/download/.
+  Visit our playground at https://www.tarantool.io/en/try-dev/
 
-To build Tarantool from source, see detailed instructions in the Tarantool
-documentation at https://tarantool.io/en/doc/2.1/dev_guide/building_from_source/.
+* Use a Docker image:
 
-Please report bugs at https://github.com/tarantool/tarantool/issues
-We also warmly welcome your feedback in the discussion mailing
-list, tarantool@googlegroups.com.
+  ```bash
+  docker run -d -p 3301:3301 -v $(pwd):/var/lib/tarantool tarantool/tarantool:2
```

Comment:
`:2` and `:latest` are now the same, so we can omit a tag (`:latest` is default).